packages/office-ui-fabric-react/src/components/Image/Image.styles.ts

```diff
@@ -77,28 +82,43 @@ export const getStyles = (props: IImageStyleProps): IImageStyles => {
       isCenter && [classNames.imageCenter, ImageFitStyles],
       isContain && [
         classNames.imageContain,
-        isLandscape && {
-          width: '100%',
-          height: 'auto'
-        },
-        !isLandscape && {
-          width: 'auto',
-          height: '100%'
-        },
-        ImageFitStyles
-      ],
-      isCover && [
-        classNames.imageCover,
-        isLandscape && {
-          width: 'auto',
-          height: '100%'
-        },
-        !isLandscape && {
+        supportsObjectFit && {
           width: '100%',
-          height: 'auto'
+          height: '100%',
+          objectFit: 'contain'
         },
+        !supportsObjectFit &&
+          isLandscape && {
+            width: '100%',
+            height: 'auto'
+          },
+        !supportsObjectFit &&
+          !isLandscape && {
+            width: 'auto',
+            height: '100%'
+          },
         ImageFitStyles
       ],
+      isCover &&
+        isCover && [
```

Comment:
isCover is used redundantly for this case

Reply:
Just wanted to be extra sure :)

Reply:
Fixed in 70fc99d